### DISCUSSION TITLE
Displaced my grass

### DISCUSSION THREAD

**Discussion by Spahbed** [visualdesigncafe/nature-renderer#20] [Help]
I installed nature render and added the component to my terrain but it scatters all my grass if ticked on to render teh grass. How do you fix this?

<img width="350" alt="Screenshot 2024-02-29 225100" src="https://github.com/visualdesigncafe/nature-renderer/assets/36801304/5132c2cd-246c-4116-b7f8-0f4b1b909c0d">
<img width="399" alt="Screenshot 2024-02-29 225111" src="https://github.com/visualdesigncafe/nature-renderer/assets/36801304/3193a184-2535-4019-bcec-9c762bb40fa6">

**Comment by Spahbed** [comment]
Also in the demo scene i cant even erase details they flciker and come back; so cant place vegetation down. Does this plugin conflict with something?

https://streamable.com/iwaw96

Video here; on the demo; I click to remove the vegetation it disspears for the whole area and tehn comes straight back with different arrangement; is this how nature renderer is intended to work ie you cant place patches where you want and its all placed semi reandomly?

**Comment by maxplooi** [comment]
It looks like your grass prefab may have an offset, resulting in wrong positions. Can you check that your grass prefab has its transform at 0,0,0? There should be a warning notice for that prefab in the Nature Renderer component as well. The warning is: _Prefab root position is not at (0,0,0)_. 

Any position that is set in the prefab's transform will be applied as offset to the rendered instances, this is mainly so that you can add a small Y offset to sink objects such as trees a bit into the ground.

**Reply by Spahbed** [reply]
Yep that fixed it; I noticed that warning before and tried to change it but it wasnt saving in the prefab, so I left it. Thanks!

**Reply by Spahbed** [reply]
Is there any way to support shadow proxies; ie turn the LODs shadows off and use just a imposter to render the shadow of the terrain details?
